Composer/packages/client/src/pages/qna/index.tsx

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useContext, Fragment, useMemo, useCallback, Suspense, useEffect } from 'react';
+import formatMessage from 'format-message';
+import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
+import { RouteComponentProps, Router } from '@reach/router';
+
+import { LoadingSpinner } from '../../components/LoadingSpinner/LoadingSpinner';
+import { StoreContext } from '../../store';
+import { actionButton } from '../language-understanding/styles';
+import { navigateTo } from '../../utils';
+import { TestController } from '../../components/TestController/TestController';
+import { INavTreeItem } from '../../components/NavTree/NavTree';
+import { Page } from '../../components/Page/Page';
+
+import TableView from './table-view';
+const CodeEditorWrapper = React.lazy(() => import('./codeEditorWrapper'));
+
+interface LGPageProps extends RouteComponentProps<{}> {
+  dialogId?: string;
+}
+
+const LGPage: React.FC<LGPageProps> = (props) => {
```

Comment:
Naming - it's QnAPage instead of LGPage?

Reply:
@liweitian  Can u see it?